package.json

```diff
@@ -71,7 +71,10 @@
 		"storybook": "start-storybook  -c ./storybook -p 6007 --ci",
 		"build-storybook": "build-storybook  -c ./storybook",
 		"changelog": "node ./bin/changelog --changelogSrcType='ZENHUB_RELEASE'",
-		"bump-version": "npm run -s install-if-deps-outdated && php ./bin/update-version.php"
+		"bump-version": "npm run -s install-if-deps-outdated && php ./bin/update-version.php",
+		"predocker-phpunit:install": "docker-compose -f docker-compose-phpunit.yml down && docker volume rm -f woocommerce-admin_testsuite",
```

Comment:
I wonder if its fast enough to do this that we could just wrap up the running of all these scripts into a single bash script or something? just to keep the cognitive overhead of running these tests in docker low?

Reply:
It'd be so cool if non docker uses tried running a single script like this and the speed/convenience got some converts to docker 😂

Reply:
Nice idea!

It takes a few minutes on my machine to complete the install script. However I think in a single script, we could detect if installation has been run- and if not then automatically start the installation before it proceeds with the tests 🤔

Reply:
@samueljseay What do you think about addressing this in a follow-up PR in the next cooldown instead?

Reply:
Yep sounds good 👍